packages/jellyfish-api-core/__tests__/category/account/listBurnHistory.test.ts

```diff
@@ -0,0 +1,195 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { BalanceTransferPayload, DfTxType } from '../../../src/category/account'
+import { createToken, accountToAccount } from '@defichain/testing'
+
+describe('Account', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+  const burnAddress = 'mfburnZSAM7Gs1hpDeNaMotJXSGA7edosG'
+  const burnAddressPrivateKey = '93ViFmLeJVgKSPxWGQHmSdT5RbeGDtGW4bsiwQM2qnQyucChMqQ'
+  let fundedAddress: string
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+
+    await client.wallet.importPrivKey(burnAddressPrivateKey)
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should listBurnHistory after test masternode creation fee burn', async () => {
+    const newAddress = await container.getNewAddress('', 'legacy')
+    await client.masternode.createMasternode(newAddress)
+    await container.generate(1)
+
+    const history = await client.account.listBurnHistory()
+    expect(history.length).toStrictEqual(1)
+    expect(history[0].owner.slice(0, 16)).toStrictEqual('6a1a446654784301')
+    expect(typeof history[0].blockHash).toStrictEqual('string')
+    expect(history[0].blockHash.length).toStrictEqual(64)
+    expect(history[0].type).toStrictEqual('CreateMasternode')
+    expect(history[0].txn).toStrictEqual(2)
+    expect(typeof history[0].txid).toStrictEqual('string')
+    expect(history[0].txid.length).toStrictEqual(64)
+    expect(typeof history[0].amounts).toStrictEqual('object')
+    expect(history[0].amounts.length).toStrictEqual(1)
+    expect(history[0].amounts[0]).toStrictEqual('1.00000000@DFI')
+  })
```

Comment:
Tests are failing when i run them independently. Each test most be independent and must not rely on other tests  to pass. Make sure your tests are passing independently.

Reply:
@Jouzo rmb to address this

Reply:
Done, let me know if this new version is coherent with the rest